package.json

```diff
@@ -61,4 +61,4 @@
     "webpack": "^5.89.0",
     "webpack-cli": "^5.1.4"
   }
-}
+}
```

Comment:
remove package.json and lockfile changes

Reply:
Fixed this, not sure why it was showing this as a diff it was exactly the same...